### ISSUE TITLE
Quoting of empty TXT records

### ISSUE THREAD

**Issue by patrick1024** [dnsjava/dnsjava#254]
Hi,

I'm not sure if this is a bug, but it is at least a different behaviour of creating empty TXT records while using the `TXTRecord` constructor or the static method `fromString` of `Record`:

```
        Record r0 = Record.fromString(recordName, Type.TXT, DClass.IN, 0, "", zoneName);
        TXTRecord r1 = new TXTRecord(recordName, DClass.IN, 0, "");
        System.out.println("r0: " + r0);
        System.out.println("r1: " + r1);
```
The output of this code snippet is:
```
r0: name.name.		0	IN	TXT
r1: name.name.		0	IN	TXT	""
```
BIND and PowerDNS servers are not able to load records with representation r0.

Patrick

**Comment by ibauersachs** [comment]
Yes, that's a bug. Thanks for the report!

**Comment by nresare** [comment]
A fix to this issue was merged into the `release/3.5.x` branch yesterday. Can you confirm that the change resolves your issue @patrick1024 ?

**Comment by patrick1024** [comment]
Yes, the change resolves my issue. Thanks @nresare
